app/views/devise/sessions/new.html.erb

```diff
@@ -0,0 +1,18 @@
+<h2 class='sign-in-message'>Log in</h2>
+<%= form_for(resource, as: resource_name, url: session_path(resource_name)) do |f| %>
+  <div class="field">
+    <%= f.email_field :email, placeholder: "Email", autofocus: true, autocomplete: "email" %>
+  </div>
+
+  <div class="field">
+    <%= f.password_field :password, placeholder: 'Password', autocomplete: "current-password" %>
+  </div>
+
+  <div class="actions">
+    <%= f.submit "Log in" %>
+  </div>
+<% end %>
+
+<div class='shared-links-css'>
+<%= render "devise/shared/links" %>
+</div>
```

Comment:
Great work done @angeluray 

- [ ] Please modified your login and registration to match the [wireframe](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/sneak_peek.md) styling as per project requirement.
![image](https://user-images.githubusercontent.com/55596970/206354097-3969a5a3-96da-4ac2-914b-cbe9b989af5f.png)